fluent-langneg/data/likelySubtags-min.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I don't know how small it will be yet. I'll need to generate the data for the `min` variant, and so far it's just a stub. I agree we'll want to bundle it, but I'd like to make it a conditional bundling i.e. when you're building fluent-langneg I'd like you to be able to choose if you want with `min` or full likelySubtags.